### PR TITLE
Add a GitHub Action workflow to release a new version

### PR DESCRIPTION
**Pull request by ste93cry**
As per title, this workflow comes straight from the repo of the Javascript SDK and should enable releasing a new version without depending on @HazAT

### CODE REVIEW

**Comment by Jean85** [comment]
Woha, this seems great! How much stuff is done by that action? And how would it be triggered?

**Comment by ste93cry** [comment]
>How much stuff is done by that action?

[This](https://github.com/getsentry/action-prepare-release) is the action, I had a quick look yesterday at the code and it seemed to me that it does everything using the Craft tool under the hood, so it should be able to automatically tag the branch and publish the release

>And how would it be triggered?

This is the only screenshot that @HazAT posted on Discord to show how it works. Basically, it should be sufficient to merge this PR and then when we want to release a new version go to the Actions tab and trigger the workflow manually by specifying the version to release and from which branch to start it

immagine

**Review by Jean85 — APPROVED** [review]
That's great! Can't wait to port it to `sentry-symfony` too!

**Comment by ste93cry** [comment]
Mmm, I noticed in the action that it looks like it doesn't publish the version, but it rather open an issue that once merged unlock the release. @HazAT am I wrong?

https://github.com/getsentry/action-prepare-release/blob/d4e63a91f26f0ad81ab25f38ebd8f3f172128870/action.yml#L86-L88

**Comment by HazAT** [comment]
@ste93cry That is correct, someone of the release managers at Sentry need to approve. But a simple ping is enough and this process is quicker and we have a track record.

**Review by stayallive — APPROVED** [review]
Looks like a great addition 👍

**Comment by ste93cry** [comment]
>someone of the release managers at Sentry need to approve

Understood, I was hoping we could became independent in releasing but this is anyway an improvement over now. Do we have to keep pinging you on Discord or is there anyone else to bother?

**Comment by HazAT** [comment]
In theory, there are notification but I can't keep up ^^
A ping in discord is simple, it's really not more than a click for me then (way better than before)

**Comment by Jean85** [comment]
Oook so I'll port this to `sentry/sentry-symfony` too! :tada:

**Pull request merged**
